app/waf/src/controllers/adc.controller.ts

```diff
@@ -344,4 +347,24 @@ export class AdcController {
       }
     });
   }
+
+  private async cUserdata(): Promise<string> {
+    let root_password: string = Math.random()
+      .toString(36)
+      .slice(-8);
+    let admin_password: string = Math.random()
+      .toString(36)
+      .slice(-8);
+
```

Comment:
user_data content looks straightforward. although I doubt we really want a random password.

Reply:
I can change it if all of you are happy to change.